### ISSUE TITLE
Watch .ghci and .ghcid

### ISSUE THREAD

**Issue by yairchu** [ndmitchell/ghcid#141]
Example scenario: pull the hlint repo and within VS code pick "Start Ghcid". The terminal opens, shows some stuff, disappears and the user gets a notification: "Error: The terminal process terminated with exit code: 1".

I think that it would be more nicer if the `ghcid` process would keep running, display an error (and perhaps assign it to `.ghcid:1`), and watch for changes in `.ghcid` and `.ghci`. Then it would reload when these change.

**Comment by ndmitchell** [comment]
#127 covers .ghci, but I agree adding `.ghcid` too is a great idea. Your idea is a good one!

**Comment by ndmitchell** [comment]
I now restart if the `.ghcid` file reloads, and I restart right from the beginning, so its effects are applied.

However, reading more closely, this ticket is about something more subtle - leaving the console open and watch for `.ghcid` on failure to start. I'm less sure about that - since people might think they started `ghcid` but really it's in a totally invalid state. What if the VS Code didn't rip the window away when it failed to start, but left the error message intact?

**Comment by yairchu** [comment]
> What if the VS Code didn't rip the window away when it failed to start, but left the error message intact?

Yes this might be the better solution. It's probably a simple change for the VS Code plugin.
